### PR TITLE
Implement cumulativeEnergyCharged for Ioniq + minor changes

### PR DESCRIPTION
**Pull request by noradtux**

### CODE REVIEW

File: cars/IONIQ_BEV.py [hunk 1]
```diff
@@ -49,11 +49,12 @@ def getData(self):
                     'dcBatteryPower':           dcBatteryCurrent * dcBatteryVoltage / 1000.0,
                     'dcBatteryVoltage':         dcBatteryVoltage,
                     'soh':                      int.from_bytes(raw[2105][0x7EC24][0:2], byteorder='big', signed=False) / 10.0,
+                    'cumulativeEnergyCharged':  int.from_bytes(raw[2101][0x7EC25][6:7] + raw[2101][0x7EC26][0:3], byteorder='big', signed=False) / 10.0,
+                    'cumulativeEnergyDischarged': int.from_bytes(raw[2101][0x7EC26][3:7], byteorder='big', signed=False) / 10.0,
```
Comment (GPlay97) [inline cars/IONIQ_BEV.py:53]: I think on discharged is the other byte missing that should be concated, isn't it?

**Review by GPlay97 — CHANGES_REQUESTED** [review]

File: cars/IONIQ_BEV.py (hunk 1 above)
Comment (noradtux) [reply cars/IONIQ_BEV.py:53]: The 4 bytes of discharged are located in a single row (bytes 4 to 8)

Comment (GPlay97) [reply cars/IONIQ_BEV.py:53]: Ah, i see, thank you :+1:

**Review by GPlay97 — APPROVED** [review]
Thank you very much!

**Pull request merged**